app/views/includes/upscan_file_upload_form.scala.html

```diff
@@ -1,48 +0,0 @@
-@*
```

Comment:
can we please document/flag somewhere that this file should be deleted prior to final merge/release

Reply:
on list for clean up